examples/todomvc/playwright.config.ts

```diff
@@ -31,7 +31,7 @@ export default defineConfig({
   workers: process.env.CI ? 1 : undefined,
 
   /* Reporter to use. See https://playwright.dev/docs/test-reporters */
-  reporter: 'html',
+  reporter: [['html'], ['list']],
```

Comment:
Why this change?

Reply:
I'd like to better test colors in the reporter output.